src/libraries/tests.proj

```diff
@@ -519,7 +514,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <SmokeTestProject Include="$(MSBuildThisFileDirectory)System.Runtime\tests\System.Runtime.Tests.csproj" />
+    <SmokeTestProject Condition="'$(UseNativeAOTRuntime)' != 'true'" Include="$(MSBuildThisFileDirectory)System.Runtime\tests\System.Runtime.Tests.csproj" />
```

Comment:
Why is this needed? Aren't these changes only related to enabling functional tests?

Reply:
Functional tests are executed together with the library tests. This should be reverted once the we enable library tests with Native AOT.